pymc/ode/__init__.py

```diff
@@ -1,4 +1,8 @@
-"""This submodule contains tools used to perform inference on ordinary differential equations."""
+"""
+This submodule contains tools used to perform inference on ordinary differential equations.
+Due to the nature of the model (as well as included solvers), this may perform slowly.
+A library based on PyMC--sunode--has implemented Adams' method and BDF (backward differentation formula).  More information about sunode is available at: https://github.com/aseyboldt/sunode.
```

Comment:
```suggestion
Another library based on PyMC--sunode--has implemented Adams' method and BDF (backward differentation formula) using the very fast SUNDIALS suite of ODE and PDE solvers.
It is much faster than the ``pm.ode`` implementation.
More information about ``sunode`` is available at: https://github.com/aseyboldt/sunode.
```
we can do one sentence per line, because the line breaks here will be ignored by Sphinx

Reply:
I have made the changes and will re-issue PR